internal/fixture/state.go

```diff
@@ -182,8 +201,22 @@ var _ Operation = &Apply{}
 
 func (a Apply) run(state *State) error {
 	err := state.Apply(a.Object, a.APIVersion, a.Manager, false)
-	if (err != nil || a.Conflicts != nil) && !reflect.DeepEqual(err, a.Conflicts) {
-		return fmt.Errorf("expected conflicts: %v, got %v", a.Conflicts, err)
+	if err != nil && err.(merge.Conflicts) == nil {
```

Comment:
Did you really want to check for the cast to be nil?
[This will panic when the error is not a conflict](https://golang.org/ref/spec#Type_assertions) instead of returning err.

Instead the ok return should be used: 
```suggestion
	if _, ok := err.(merge.Conflicts); err != nil && !ok {
```

This would as far as I know cast on every pass which might not be wanted, so probably better wrap this?
```suggestion
	if err != nil {
	  if _, ok := err.(merge.Conflicts); !ok {
```